app-shells/fzf/fzf-0.21.0.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit go-module bash-completion-r1
```

Comment:
You can use an eclass for `zsh` too.

Reply:
I can't fins zsh eclass listed here: 
https://devmanual.gentoo.org/eclass-reference/